templates/default/views/DashboardView.html

```diff
@@ -26,30 +26,42 @@
             </a>
         </li>
     </ul>
-    <input 
-        type="search" 
-        class="form-control" 
-        ng-model="query" 
-        uib-typeahead="name as tc.name for tc in testCases | filter:$viewValue | limitTo:8" 
-        typeahead-on-select="setQueryFilterValue($item.name)" 
-        placeholder="Test Case Filter" 
-    />
+      <div class="form-horizontal">
+          <div class="form-group">
+            <div class="col-sm-10">
+              <input 
+                 type="search" 
+                 class="form-control" 
+                 ng-model="query" 
+                 uib-typeahead="name as tc.name for tc in testCases | filter:$viewValue | limitTo:8" 
+                 typeahead-on-select="setQueryFilterValue($item.name)" 
+                 placeholder="Test Case Filter" 
+              />  
+            </div>
+            <div class="col-sm-2 checkbox">
+               <label for="excludeFilter"><input type="checkbox" id="excludeFilter" ng-model="excludeFilter" ng-checked="excludeFilter"><code> Exclude</code></label>

```

Comment:
According to [angular](https://docs.angularjs.org/api/ng/directive/ngChecked) _ng-checked_ should not be used together with _ng-model_